pincer/utils/__init__.py

```diff
@@ -23,5 +24,5 @@
     "EventMgr", "HTTPMeta", "MISSING", "MissingType", "Snowflake", "Task",
     "TaskScheduler", "Timestamp", "chdir", "choice_value_types", "convert",
     "get_index", "get_params", "get_signature_and_params", "should_pass_cls",
-    "should_pass_ctx"
+    "should_pass_ctx", "replace", "construct_client_dict", "ChannelProperty", "GuildProperty"
```

Comment:
```suggestion
    "should_pass_ctx", "replace", "construct_client_dict", "ChannelProperty",
    "GuildProperty"
```

Reply:
Theres a gh action that will fix this once its pushed to main